src/src/models/Buttonmodel.ts

```diff
@@ -0,0 +1,10 @@
+interface ButtonModel {
+    customClass: string;
+    buttonIcon: string;
+    disabled: boolean;
+    buttondName: string;
+    value: string;
+    readOnly: boolean;
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
}
```